mloop_interface.py

```diff
@@ -22,10 +22,13 @@ def set_globals_mloop(mloop_session=None, mloop_iteration=None):
 
 class LoopInterface(Interface):
     def __init__(self):
-        super(LoopInterface, self).__init__()
-
         # Retrieve configuration from file or generate from defaults
         self.config = mloop_config.get()
+
+        # Pass config arguments to parent class's __init__() so that any logging

```

Comment:
Even though [`mloop.Interface`](https://github.com/michaelhush/M-LOOP/blob/87aa82a90e4da346402523af96e4bbe7311dd945/mloop/interfaces.py#L74) only has one keyword argument (`interface_wait`) that isn't passed to [`mloop.utilities._config_logger`](https://github.com/michaelhush/M-LOOP/blob/87aa82a90e4da346402523af96e4bbe7311dd945/mloop/utilities.py#L50), perhaps omit "logging" from this comment.

Reply:
Sounds good, edited in 745611f.